app/Domains/Settings/ManageGiftOccasions/Services/UpdateGiftOccasionPosition.php

```diff
@@ -62,13 +61,10 @@ private function validate(): void
     {
         $this->validateRules($this->data);
 
-        $this->giftOccasion = GiftOccasion::where('account_id', $this->data['account_id'])
+        $this->giftOccasion = $this->account()->giftOccasions()
             ->findOrFail($this->data['gift_occasion_id']);
 
-        $this->pastPosition = DB::table('gift_occasions')
-            ->where('id', $this->giftOccasion->id)
-            ->select('position')
-            ->first()->position;
+        $this->pastPosition = $this->giftOccasion->position;
```

Comment:
@djaiss no need to use `DB::table`, the position is a property of the `giftOccasion` object